pandas/tests/reshape/test_pivot.py

```diff
@@ -19,6 +19,17 @@ def dropna(request):
     return request.param
 
 
+@pytest.fixture(params=['right', 'left'])
+def closed(request):
```

Comment:
There is a `closed` fixture in the top level `conftest.py` that's generally used with `IntervalIndex` that you can use instead.  It also includes `'both'` and `'neither'` as parameters.

Reply:
I can't find it in
pandas/tests/indexing/conftest.py
or
pandas/tests/indexes/conftest.py
Can you give me a path to it ?

Reply:
It's in `pandas/conftest.py`:
https://github.com/pandas-dev/pandas/blob/129aaea46db94d5f8296bd5daeff9dc124b1a99e/pandas/conftest.py#L337-L342

No need to import it or anything though; if you just delete the your current `closed` fixture `pytest` should automatically pick it up.